README.md

```diff
@@ -1,81 +1,106 @@
-- [nVidia CUDA](https://developer.nvidia.com/cuda-zone) (if you want to use GPU)
-- [python](https://www.python.org/), [numpy](http://www.numpy.org/) (if you want to use the integrated python interpreter)
-- [python](https://www.python.org/), [sympy](http://www.sympy.org/) and R package: [rPython](https://cran.r-project.org/package=rPython) (if you want to develop a model using python instead of R)
-- [MPI](https://en.wikipedia.org/wiki/Message_Passing_Interface) (e.g. [OpenMPI](http://www.open-mpi.org/))
+### Documentation
 
-You can install many of these with the tools/install.sh script (note it requires sudo):
+The documentation (including tutorials) is published at
+[docs.tclb.io](https://docs.tclb.io/).
 
-```bash
-sudo tools/install.sh cuda 6.5-14 # only if your GPU supports cuda
-sudo tools/install.sh r
-sudo tools/install.sh openmpi
-     tools/install.sh rdep
-sudo tools/install.sh python-dev
-     tools/install.sh rpython
-sudo tools/install.sh module # only on CentOS
-```
+For the `develop` version, most recent documentation can be found at
+[develop.docs.tclb.io](https://develop.docs.tclb.io/).
+
+You can contribute to the documentation at
+[CFD-GO/TCLB_docs](https://github.com/CFD-GO/TCLB_docs).
+
+### Supported architectures
+This code is designed to run on **Linux**. We strongly recommend using Linux for compilation, computation and postprocessing.
+
+Nevertheless, TCLB can be compiled on Windows (CPU only), using the [Windows Subsystem for Linux](https://pl.wikipedia.org/wiki/Windows_Subsystem_for_Linux). It also can be compiled on MacOS (also CPU only). Both Debian and Red Hat based linux distributions are supported by the `install.sh` script described below, as is MacOS (with `brew` package manager).
 
-The `install.sh` script is designed to work on Ubuntu (e.g. on the [Travis-CI](https://travis-ci.org/CFD-GO/TCLB) VMs). 
-The `install.sh` script should work on CentOS.
-You can install the **`sudo`** parts by yourself, and use script to install R packages: rdep and rpython.
+### Dependencies
 
+For the code to compile and work you'll need a few things:
+- [R](https://www.r-project.org/) and some R packages ([optparse](https://cran.r-project.org/package=optparse), [numbers](https://cran.r-project.org/package=numbers), [rtemplate](https://github.com/llaniewski/rtemplate), [gvector](https://github.com/llaniewski/gvector), [polyAlgebra](https://github.com/llaniewski/polyAlgebra))
+- [MPI](https://en.wikipedia.org/wiki/Message_Passing_Interface). We recommend [OpenMPI](http://www.open-mpi.org/)
+- If you want to use your GPU, you'll need [nVidia CUDA](https://developer.nvidia.com/cuda-zone)
+- If you want to integrate TCLB with Python, you'll need [python](https://www.python.org/), [numpy](http://www.numpy.org/) with headers and stuff
```

Comment:
I wouldn't say "and stuff" at the end. Possibly replace with "with headers etc."